src/components/ProductLink/styles.module.scss

```diff
@@ -1,6 +1,4 @@
 .productWrapper {
-  width: 200px;
```

Comment:
Good on making the component responsive width but unfortunately it affects how it is rendered in storybook. Check out how @sprajams handled the same issue for the collection grid here: https://github.com/RealSpiffy/shop/pull/11/files